source/hands_on/sqil.rst

```diff
@@ -207,7 +207,7 @@ we use a well-trained model to generate data in the collecting stage and push th
 
 We also need to modify rewards for new data and demonstration data. Taking the CartPole environment as an example:
 
- .. code-block:: python
+.. code-block:: python
 
         new_data = collector.collect_data(learner.train_iter, policy_kwargs={'eps': eps})
         for i in range(len(new_data)):
```

Comment:
In line 217 there is a typo: Regarding

Reply:
Also in line 219

Reply:
What about changing 'we can leave these rewards as they were' with 'we can leave these rewards unchanged'

Reply:
in 219: , the table below shows a benchmark of the performance of DQN, SQL, and SQIL in Lunarlander and Pong environments.

Reply:
In 242: Pong, not pong